### PR TITLE
Presence | DevTR

### PR DESCRIPTION
**Pull request by eren885**
image
image

### CODE REVIEW

**Review by Bas950 — CHANGES_REQUESTED** [review]
upload proof of it working

**Comment by eren885** [comment]
its been 3 hours and deepscan still analyzing it out, bruh

File: websites/D/DevTR/presence.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    if (page.startsWith("/kullanici/")) {
+        presenceData.details = "Bir kullanıcının profilini görüntülüyor:";
+        presenceData.state = document.querySelector("body > div:nth-child(6) > div > h3").textContent;
+    } else if (page.startsWith("/hakkimizda")) {
+        presenceData.details = "Bir sayfayı görüntülüyor:";
+        presenceData.state = "Hakkımızda";
+    } else if (page.startsWith("/javascript")) {
+        presenceData.details = "Bir sayfayı görüntülüyor:";
+        presenceData.state = "JS Komutlar";
+        presenceData.smallImageKey = "js"
```
Comment (SeMiD) [inline websites/D/DevTR/presence.ts:21]: Codacy found an issue: [Missing semicolon.](https://app.codacy.com/gh/PreMiD/Presences/pullRequest?prid=6113506)

**Comment by eren885** [comment]
ah

**Review by doomlerd — APPROVED** [review]

File: websites/D/DevTR/presence.ts [hunk 2]
```diff
@@ -0,0 +1,51 @@
+const presence = new Presence({
+    clientId: "749319733807153162"
+});
+presence.on("UpdateData", () => {
+    const presenceData: PresenceData = {
+        largeImageKey: "logo"
+    },
+     page = window.location.pathname,
+    browsingStamp = Math.floor(Date.now() / 1000);
+  //a
```
Comment (Bas950) [inline websites/D/DevTR/presence.ts:10]: remove the random comment

**Review by Bas950 — APPROVED** [review]

**Review by doomlerd — APPROVED** [review]

**Pull request merged**